eth_tester/backends/mock/factory.py

```diff
@@ -259,6 +259,15 @@ def make_genesis_block():
         "uncles": [],
     }
 
+    if overrides is not None:
+        if not (all(bool(override in genesis_block) for override in overrides)):
```

Comment:
It takes a litlte extra work but I think we can improve the error message below and the readability of this code.

```python
allowed_fields = set(genesis_block.keys())
override_fields = set(overrides.keys())
unexpected_fields = tuple(sorted(override_fields.difference(allowed_fields)))
if unexpected_fields:
    raise ValueError(
        "The following invalid fields were supplied to override the genesis parameters: {0}.".format(unexpected_fields)
    )
```

```